styles/chess.com/catppuccin.user.css

```diff
@@ -16,22 +16,14 @@
 ==/UserStyle== */
 
 @-moz-document domain("chess.com") {
-  body {
+  body.light-mode {
     #catppuccin(@lightFlavor, @accentColor);
   }
 
-  .dark-mode {
+  body.dark-mode {
     #catppuccin(@darkFlavor, @accentColor);
   }
```

Comment:
i belive this was done like this since there is multiple applications places for .dark-mode.